docs/includes/generated_docs/schemas/beta.tpp.md

```diff
@@ -521,7 +537,16 @@ referrals are recorded in the clinical events table but this data will be incomp
 <p class="dimension-indicator"><code>many rows per patient</code></p>
 ## ec
 
+Emergency care attendances data — the Emergency Care Data Set (ECDS) —
+is provided via the NHS Secondary Uses Service.
+
+This table gives core details of attendances.
+
+Refer to the [OpenSAFELY documentation on the ECDS data source][ecds_data_source_docs]
+and the GitHub issue that [discusses more of the background context][ecds_context_issue].
 
+[ecds_data_source_docs]: https://docs.opensafely.org/data-sources/ecds/
+[ecds_context_issue]: https://github.com/opensafely-core/cohort-extractor/issues/182
```

Comment:
question: As the APCS reference information, should we only include the reference links in the `ec` table or for each ECDS-related table?